custom-recipes/named-entity-recognition-extract/recipe.py

```diff
@@ -48,24 +48,20 @@
 
 if ner_model == "spacy":
     from ner.spacy import extract_entities
-
-    language = recipe_config.get("text_language_spacy", "en")
+    model_id = recipe_config.get("text_language_spacy", "en")
 else:
-    from flair.models import SequenceTagger
+    import flair
     from ner.flair import extract_entities
-    
-    tagger = SequenceTagger.load("flair/ner-english-fast@3d3d35790f78a00ef319939b9004209d1d05f788")
+    model_id = recipe_config.get("text_language_flair", "en")
```

Comment:
I chose to use `model_id` rather than `language` since flair handles two english model (and it's much more flexible anyway)